### PR TITLE
feat(ids): generate library

### LINKED ISSUE #7912: Cloud IDS API: `ids.googleapis.com`

### PR DESCRIPTION
**Pull request by devbww**
Fixes #7912.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/7920)

### CODE REVIEW

File: google/cloud/ids/README.md [hunk 1]
```diff
@@ -3,7 +3,12 @@
 :construction:
 
 This directory contains an idiomatic C++ client library for the
-[Cloud IDS API][cloud-service-docs], a service to Cloud IDS (Cloud Intrusion Detection System) detects malware, spyware, command-and-control attacks, and other network-based threats. Its security efficacy is industry leading, built with Palo Alto Networks technologies. When you use this product, your organization name and consumption levels will be shared with Palo Alto Networks.
+[Cloud IDS (Intrusion Detection System) API][cloud-service-docs], a
```
Comment (dbolduc) [inline google/cloud/ids/README.md:6]: I think acronyms are typically: "Intrusion Detection System (IDS)"... then just "IDS" thereafter.

File: google/cloud/ids/doc/main.dox [hunk 2]
```diff
@@ -2,8 +2,13 @@
 
 @mainpage Cloud IDS API C++ Client Library
 
-An idiomatic C++ client library for the [Cloud IDS API][cloud-service-docs], a service
-to Cloud IDS (Cloud Intrusion Detection System) detects malware, spyware, command-and-control attacks, and other network-based threats. Its security efficacy is industry leading, built with Palo Alto Networks technologies. When you use this product, your organization name and consumption levels will be shared with Palo Alto Networks.
+An idiomatic C++ client library for the
+[Cloud IDS (Intrusion Detection System) API][cloud-service-docs], a
```
Comment (dbolduc) [inline google/cloud/ids/doc/main.dox:6]: ditto

File: external/googleapis/update_libraries.sh [hunk 3]
```diff
@@ -54,6 +54,7 @@ declare -A -r LIBRARIES=(
       "@com_google_googleapis//google/iam/credentials/v1:credentials_proto" \
       "@com_google_googleapis//google/iam/admin/v1:admin_proto"
   )"
+  ["ids"]="@com_google_googleapis//google/cloud/ids/v1:ids_cc_grpc"
```
Comment (dbolduc) [inline external/googleapis/update_libraries.sh:57]: I forget where we landed on this. I am just commenting for the sake of commenting.

**Review by dbolduc — APPROVED** [review]

File: google/cloud/ids/README.md (hunk 1 above)
Comment (devbww) [reply google/cloud/ids/README.md:6]: We were just copying the text from the documentation summary in the service yaml file.

But that said, I agree with you and I'll change it when I regenerate this code after #7926 lands.  Thanks.

File: external/googleapis/update_libraries.sh (hunk 3 above)
Comment (devbww) [reply external/googleapis/update_libraries.sh:57]: I think we landed on using the `_cc_grpc` form as easier/preferred.

File: google/cloud/ids/README.md (hunk 1 above)
Comment (devbww) [reply google/cloud/ids/README.md:6]: Oops ... no ... I got confused between "ids" and "iot".  :-)

I'll correct this here/now.

Comment (devbww) [reply google/cloud/ids/README.md]: Done.

File: google/cloud/ids/doc/main.dox (hunk 2 above)
Comment (devbww) [reply google/cloud/ids/doc/main.dox]: Done.

**Pull request merged**